src/compiler/crystal/syntax/parser.cr

```diff
@@ -6336,6 +6354,15 @@ module Crystal
       @temp_arg_count += 1
       arg_name
     end
+
+    def consume_statement_end
```

Comment:
When expecting a statement end, it may as well be more than one. So this method should probably directly skip them all. `consume_whitespace` and `consume_newlines` do the same.

Every call to this method is immediately followed by a call to `skip_statement_end` already, so we could easily pull this in.